case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+7. Some features that are commonly used in software development may not be available in Azure DevOps, or may require additional tools or services to be used.
+
+## **Pricing**
+
+The pricing of using Azure DevOps depends on the size of your team or organization. The image below shows the pricing model that Azure DevOps have.
+
+<div style="width: 600px;">
+
+![Price Image](https://i.ibb.co/5jCdgMn/image.png)
+
+<p style="padding: 0; margin: 0;">
+
+*Source: Azure DevOps homepage*
+
+</p>
+
+</div>
+
+As you can see in the image above there is a wide range of options to choose from when it comes to the services that Azure DevOps have to offer.
+
+The **Basic plan** of Azure DevOps is a free plan that provides limited features for small teams or individual developers. You will have access to the following features with basic plan:
+
+1. Version control, you can use Git to manage your source code
+2. Work Tracking by using the project boards. You can create work items to track issues, bugs, tasks and backlogs to prioritize and manage work.
+3. Package management.
+4. Basic security and compliance: You can use features such as pull requests and code reviews to secure your code and compliance features such as auditing and disaster recovery to meet regulatory requirements.
+5. Continuous integration (CI): You can set up CI pipelines to automatically build, test and deploy your code.
+(*There are some limitations with the Basic plan, such as the number of parallel pipelines and the number of minutes available for CI/CD, but it is a great option for small teams or individual developers who are just getting started with DevOps.*)
+
+### **Support**
+
+There is an option for premium support from Azure DevOps.
+The cost of premium support for Azure DevOps varies based on a number of factors, including the level of support required, the number of users and the level of service desired.
+
+Microsoft does not publicly disclose the exact cost of its premium support plans for Azure DevOps, as pricing is customized for each organization based on its specific needs and requirements. To obtain an accurate estimate of the cost of premium support for Azure DevOps, organizations should contact Microsoft directly to discuss their specific needs and requirements.
+
+It is worth noting that while premium support may have a higher upfront cost than the basic support, it can provide organizations with a number of benefits, including faster response times, dedicated support engineers, access to a wider range of support resources and expertise, which can help to minimize downtime and ensure that any issues are resolved quickly and effectively.
+
+## **Comparison**
+
+While Azure DevOps is one of the leading platforms out on the marked, there are more options for the developers to use. Here is some alternative to Azure DevOps.
+
+<table style="width:100%;">
```

Comment:
https://www.tablesgenerator.com/markdown_tables#

Please format this as a markdown table

Reply:
You should not style the table, format styling is the responsibility of the app that renders the markdown

Reply:
Is this a must to have it as a MD table? I personally don't like how it looks when writing it, its a bit confusing to use.

Reply:
It's hard to use which is why I prefer a tool - but yes, you do need to use it. Markdown is built on a principle of simplicity, while this sometimes comes at the cost of usability for the author - it is the reason we selected it for use.